doc/design/cluster_train/save_model.md

```diff
@@ -0,0 +1,97 @@
+# Design Doc: Save Model
+
+## Overview
+
+The model is the output of the training process. There are two
+ways from which user can obtain a model:
+
+- Save model triggered by user code: user code asks PaddlePaddle to
```

Comment:
@typhoonzero Actually it depends on if we implement a method for saving model, or let user save model from the parameters by himself. Can you take a look at https://github.com/PaddlePaddle/Paddle/pull/2655#discussion_r124711056 ?